frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -50,6 +51,17 @@ export enum CreateApplicationFlow {
   JarUpload = 'Upload JAR file',
 }
 
+export const EditFlowPageTitle = {
+  // t('devconsole~Import from Git')
+  [CreateApplicationFlow.Git]: 'devconsole~Import from Git',
+  // t('devconsole~Import from Dockerfile')
+  [CreateApplicationFlow.Dockerfile]: 'devconsole~Import from Dockerfile',
+  // t('devconsole~Deploy Image')
+  [CreateApplicationFlow.Container]: 'devconsole~Deploy Image',
+  // t('devconsole~Upload JAR file')
+  [CreateApplicationFlow.JarUpload]: 'devconsole~Upload JAR file',
+};
```

Comment:
I'm not opposed to this model, but this puts the onus on the caller to understand these are `t` values. Seems odd to impose a dependency on a caller of a map 🤔 

Only alternative I can think of is making this a function call... `getEditFlowTitleWording(CreateApplicationFlow)` => `t` string.

Reply:
No change required... just voicing a thought.

Reply:
changed it to a util